mod-audit-server/src/main/java/org/folio/util/JsonPropertyFetcher.java

```diff
@@ -0,0 +1,51 @@
+package org.folio.util;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.joda.time.DateTime.parse;
+
+import org.joda.time.DateTime;
```

Comment:
My suggestion would be to get rid of obsolete `org.joda.time` library and use `LocalDateTime` available out of the box

Reply:
This is unified approach that used in the mod-circulation. Could you please confirm that soda DateTime -> Json date/time -> LocalDate time will work as expected?

Reply:
I see. There is no other simple ways to keep date compatible with mod-circulation format